src/explorer/gql/utils.ts

```diff
@@ -0,0 +1,27 @@
+export const BRIDGE_TXN_API_GRAPHQL_URL =
+  "https://syn-explorer-api.metagabbar.xyz/graphql";
+
+export const QUERY_INFO = {
```

Comment:
better way to do this? like an enum with strings as the value?

Reply:
I feel like `QUERY_INFO` is not telling. These are the fields we're returning from the query. So maybe, something like `QUERY_RESPONSE_FIELDS` or something makes more sense